# MPC Performance Metrics
std_msgs/Header header
int32 cav_id
float64 model_time_us
float64 solver_time_us
float64 total_time_us
int32 horizon
int32 solver_iterations
string solver_status
